v3/weaviate/graphql/aggregate.go

```diff
@@ -74,11 +74,23 @@ func (ab *AggregateBuilder) createFilterClause() string {
 	return fmt.Sprintf("(%s)", strings.Join(filters, ", "))
 }
 
+func (gb *AggregateBuilder) createFieldsClause() string {
```

Comment:
Curious about the receiver name being change to `gb` here, maybe an accident?

Reply:
yes, accident, copy / paste mistake, I'll fix it